Common/GPU/Vulkan/VulkanMemory.cpp

```diff
@@ -150,367 +130,17 @@ void VulkanPushBuffer::Unmap() {
 	if (!writePtr_)
 		return;
 
+	/*
+	// We could never hit this path before because we specified the mask explicitly.
```

Comment:
If VMA will never create this either, should we just remove?  I think it was here from some previous experimenting with non-coherent.

-[Unknown]